internal/graph/check_test.go

```diff
@@ -376,9 +397,16 @@ type document
 					t.Parallel()
 
 					checker := NewLocalChecker(
-						// TODO build this wrapper inside ResolveCheck so that we don't need to construct a new Checker per test
```

Comment:
As I'm reading it, this TODO is no longer resolvable because the check resolvers are instantiated at the server-level now. It wouldn't make sense to construct inside `ResolveCheck`. However it does appear that the `NewLocalChecker` call here might be able to be pulled outside the loop, but I haven't looked at the test itself super closely.

Reply:
@willvedd is correct. The CheckResolver is instantiated at the top-level, not per request and so the commented TODO is no longer applicable. I have moved the LocalChecker construction up a level in the test though (as Will pointed out).